developer_setup.md

```diff
@@ -34,6 +34,37 @@
       ```
 
 ### Mac
+  1. Install PostgreSQL
+    - Download and install the .dmg for 9.2.4 for MacOSX:  http://www.enterprisedb.com/products-services-training/pgdownload#osx
+    - Open the dmg and run the installer (It will update your shared memory settings and prompt for a reboot, reboot it)
+    - Open the dmg and run the installer again after the reboot.
```

Comment:
Think we're down to ~3 devs with Mac postgres (not homebrew).

I encourage everyone to go over to homebrew.
All new installs are using homebrew.